src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,399 @@
+export class TechLoad {
+
+  public jobs: JobData[];
+  public jobCount: number;
+  public uuid: string;
+
+  private seed: number;
+  private seedIndex: number = 0; // doesn't really matter what it's instantiated to, just needs to be any number >= 0 so the seed methods can generate the next number
+
+  constructor(uuid: string, amount?: number) { // Can be built with a given amount, but if not will seed the number of jobs to build
+    this.uuid = uuid;
+    this.seed = Math.abs(this.convertStrToNum(uuid));
+    
+    this.jobs = [];
+    // generate a random number of jobs, weighted towards ~3
+    const generatedJobCount = Math.round( (this.nextRand()*7 + this.nextRand()*7) / 2 );  // This line needs to run regardless of if we were passed an amount, or else the seeded rng will get messed up!
+    this.jobCount = amount ? amount : generatedJobCount; // allows for front end to request a specific number of jobs. If amount is not present, use the above generated number
+    for (let i = 0; i < this.jobCount; i++) {
+      this.jobs.push(this.generateJob()); // Where all the "magic" happens, generateJob will create all job data and push it into job array
+    }
+  }
+
+  /**
+   * @description - Called by constructor to generate a job. 
+   * @returns {JobData} - Returns an object which contains the entirety of job data required to fill out a fake ticket
+   */
+  private generateJob(): JobData {
+    // The following variables are generated prior to the output object, as they are passed into generator methods in order to generate data consistent throughout the ticket
+    const firstName = this.generateFirstName();
+    const lastName = this.generateLastName();
+    const streetAddress = this.generateStreetAddress();
+    const jobType = this.randOf(['Install', 'Repair', 'Helper', 'POTS', 'BSW']);
```

Comment:
randOf<T>()

You can pass type used here.  So 

`this.randOf<string[]>()`

Reply:
Wouldn't it be `this.randOf<string>()`? As the generic type is the same as the output, and the input is an array of the T. Also I didn't because it seems to infer it, but I will if you think it's good practice. Idk if it infers from the input or the variable into which the output is stored, but whatever the case if you hover over it: 
<img width="848" alt="Screen Shot 2023-05-20 at 10 59 45 PM" src="https://github.com/micah-wehrle/ABCNest/assets/11700686/229a692e-bea4-4899-a7ca-c4f139dcdffb">

Reply:
Inference is best ts practice. Atlas standards using implicit typing. And yah. You're right. Drop the brackets.

Reply:
I changed to definite typing